src/main/java/no/entur/antu/netexdata/collectors/ServiceJourneyActiveDatesCollector.java

```diff
@@ -0,0 +1,349 @@
+package no.entur.antu.netexdata.collectors;
+
+import jakarta.xml.bind.JAXBElement;
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+import java.util.*;
+import java.util.stream.Collectors;
+import org.entur.netex.validation.validator.jaxb.JAXBValidationContext;
+import org.entur.netex.validation.validator.jaxb.NetexDataCollector;
+import org.entur.netex.validation.validator.jaxb.support.DatedServiceJourneyUtils;
+import org.entur.netex.validation.validator.model.ServiceJourneyId;
+import org.redisson.api.RLock;
+import org.redisson.api.RedissonClient;
+import org.rutebanken.netex.model.*;
+
+public class ServiceJourneyActiveDatesCollector extends NetexDataCollector {
```

Comment:
Could you please add Javadoc at class level that briefly explain what this collector does?